### PR TITLE
Leaderboards

### PR DESCRIPTION
**Pull request by afieldi**
image

image

image

Still a few things to do. Need to add a link to the page, and automatically sort by the selected option when selecting a new filter.

### CODE REVIEW

File: Common/utils.ts [hunk 1]
```diff
@@ -129,10 +130,16 @@ export function ToGameId(matchId: string): number
   return Number(split[1]);
 }
 
-export function GameTypeToString(gameType: number, seasonId: number): string
+export function GameTypeToString(gameType: number, seasonId: number, seasons?: SeasonModel[]): string
 {
   if (seasonId)
   {
```
Comment (Vexrax) [inline Common/utils.ts:136]: bracket on newline :(

**Review by Vexrax — COMMENTED** [review]
I think the `EDIT COLUMNS` buttons need a couple changes:
1. The button size currently is slightly larger than the filter bar buttons, it should be the same size
2. I think the button should be aligned to the right.

Other than that, LGTM

**Comment by afieldi** [comment]
New 
image

File: Common/utils.ts (hunk 1 above)
Comment (afieldi) [reply Common/utils.ts:136]: Die

**Review by Vexrax — APPROVED** [review]
LGTM

**Pull request merged**